Driver/Driver/Scenes/HomeScreen/HomeViewController/HomeViewController.swift

```diff
@@ -8,14 +8,35 @@
 import UIKit.UIViewController
 
 final class HomeViewController: UIViewController {
+    // MARK: Outlets
+    @IBOutlet weak var settingsButton: CustomReusableButtonView!
+    @IBOutlet weak var roadStatusButton: CustomReusableButtonView!
+    @IBOutlet weak var maydayButton: CustomReusableButtonView!
+    @IBOutlet weak var statusButton: CustomReusableButtonView!
+    
     
     // MARK: Properties
     var viewModel: HomeViewModelProtocol!
     
     // MARK: Life Cycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        setButtons()
+    }
+    
+    private func setButtons() {
```

Comment:
Please add a screenshot from your development. We should see how buttons is shown in the screen.

Reply:
![Simulator Screen Shot - iPhone 8 Plus - 2023-03-05 at 10 37 51](https://user-images.githubusercontent.com/75314877/222948270-90405b85-302f-4185-82bd-2874689ef073.png)
I chose some random pictures based on the title. Since they are multi-colored, they can be changed .I also added a blur view to prevent the map text from mixing with the button text.